### DISCUSSION TITLE
How would bitcoinmirror enable replacing WBTC?

### DISCUSSION THREAD

**Discussion by Mckiev** [Divide-By-0/ideas-for-projects-people-would-use#28] [Q&A]
Could someone please explain? 

From https://bitcoinmirror.org/:

> Proof of burn. You burn 1 BTC. You post proof to a contract, which lets you mint 1 BBTC (burnt Bitcoin) on Ethereum 😈

It indeed creates a scarce token on ethereum chain, but it's impossible to "withdraw" from it, unlike with WBTC

Someone still has to custody the keys for wrapped BTC, to replicate WBTC, right?

**Comment by Divide-By-0** [comment]
Hey -- correct; alone, this would be a one-way bridge, meaning the destination token is worthless. One hacky way is to build the other direction on some bitcoin L2. The most robust way is to witness encrypt the destination ("burn address") private key with one time programs as proposed by Leona here: https://ethresear.ch/t/trustless-bitcoin-bridge-creation-with-witness-encryption/11953

**Reply by Mckiev** [reply]
Wow, thanks a lot for suggesting, I'll take a closer look. Trustless WBTC would be fire!
